specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AutoDismissAlertsRules/GetAutoDismissAlertsRulesWithAlertType_example.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "alertType": "IpAnomaly"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/autoDismissAlertsRules/dismissIpAnomalyAlerts",
+            "name": "dismissIpAnomalyAlerts",
+            "type": "Microsoft.Security/autoDismissAlertsRules",
+            "properties": {
+              "alertType": "IpAnomaly",
+              "lastModifiedUTC": "2019-07-31T19:50:47.083633Z",
+              "expirationDateUTC": "2019-12-01T19:50:47.083633Z",
+              "state": "Enabled",
+              "reason": "Detects a normal activity (False Positive)",
+              "comment": "Test VM",
+              "autoDismissAlertsScope": {
+                "allOf": [
+                  {
+                    "field": "CompromisedEntity",
+                    "in": [
+                      "ContosoVM1",
+                      "demoAccount@microsoft.com"
```

Comment:
>microsoft [](start = 35, length = 9)

suggesting to use "contoso.com" as it is usually the fake company used by MS demos

Reply:
done